packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts

```diff
@@ -0,0 +1,103 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { cVar, sizes } from '@/styles'
+
+type Variant = 'default' | 'compact'
+
+type NotificationWrapperProps = {
+  read?: boolean
+  selected?: boolean
+  loading?: boolean
+  variant: Variant
+}
+
+const getNotificationWrapperStyles = ({ read, selected, loading, variant }: NotificationWrapperProps) => {
+  if (selected) {
+    return css`
+      background-color: ${cVar('colorBackgroundElevated')};
+    `
+  }
+  if (!read) {
+    if (variant === 'default') {
+      return css`
+        background-color: ${cVar('colorBackground')};
+
+        :hover {
+          background-color: ${cVar('colorBackgroundStrong')};
+        }
+      `
+    }
+    return css`
+      background-color: ${cVar('colorBackgroundMutedAlpha')};
+
+      :hover {
+        background-color: ${cVar('colorBackgroundStrong')};
+      }
+    `
+  }
```

Comment:
```suggestion
  if (!read) {
    return css`
      background-color: ${variant === 'default' ? cVar('colorBackground') : cVar('colorBackgroundMutedAlpha')};

      :hover {
        background-color: ${cVar('colorBackgroundStrong')};
      }
    `
  }
```
Correct me if I'm wrong, but I think we could shorten this.